stories/tws-trends.stories.mdx

```diff
@@ -77,7 +77,7 @@ taxonomy:
 <Block>
   <Prose>
   ## Comparing the trends in water and carbon cycles
-  We applied a seasonal and trend decomposition algorithm to get the trend estimates for terrestrial water storage and gross primary production. The method can better help to deal with [nonstationarities](https://github.com/Earth-Information-System/sea-level-and-coastal-risk/blob/main/AMS_2023_Wanshu_Nie_for_VEDA_Discovery.pdf) and seasonal shifts and provide a more robust estimate of trends. These trend data sets are also provided in the [VEDA dataset page](https://www.earthdata.nasa.gov/dashboard/data-catalog).
+  We applied a seasonal and trend decomposition algorithm to get the trend estimates for terrestrial water storage and gross primary production. The method can better help to deal with [nonstationarities](https://github.com/Earth-Information-System/sea-level-and-coastal-risk/blob/main/AMS_2023_Wanshu_Nie_for_VEDA_Discovery.pdf) and seasonal shifts and provide a more robust estimate of trends. These trend data sets are also provided in the [VEDA dataset page](https://www.earthdata.nasa.gov/dashboard/data-catalog?taxonomy=%7B%22Topics%22%3A%22eis%22%7D).
```

Comment:
```suggestion
  We applied a seasonal and trend decomposition algorithm to get the trend estimates for terrestrial water storage and gross primary production. The method can better help to deal with [nonstationarities](https://github.com/Earth-Information-System/sea-level-and-coastal-risk/blob/main/AMS_2023_Wanshu_Nie_for_VEDA_Discovery.pdf) and seasonal shifts and provide a more robust estimate of trends. These trend data sets are also provided in the [VEDA dataset page](https://www.earthdata.nasa.gov/dashboard/data-catalog/lis-global-da-trends).
```

Reply:
Yeah, I'm not sure who signs off the copy of the descriptions, but I assume it's not us? Merging then, if someone raises a concern I'll tweak it again.